### DISCUSSION TITLE
Implemention of AI trainning

### DISCUSSION THREAD

**Discussion by oscarhx** [PMEAL/porespy#831] [Q&A]
hello professors,
The AI diffusion conduction prediction for pore pairs is very interesting, The trained network is applied in the example, but I meet some difficulty in coding the trainning process

**Comment by jgostick** [comment]
Hi @oscarhx
We recently had an undergrad student do a term project where they created an example on how well the AI worked on totally novel data (like a sandstone), and create [an exmample](https://porespy.org/examples/networks/tutorials/predicting_diffusive_size_factors_rock_sample.html). We were pleasantly surprized by how well the AI performed.  We were planning on also creating an example which described how to re-train or extra-train the model, but given how well it performs already this is a low priority.
